hoomd/mpcd/init.py

```diff
@@ -71,6 +71,60 @@ def make_random(N, kT, seed):
     hoomd.context.current.mpcd = data.system(_mpcd.SystemData(sysdef,pdata))
     return hoomd.context.current.mpcd
 
+def make_random_sphere(density, R, kT, seed):
+    R"""Initialize particles randomly inside a sphere
+
+    Args:
+        density (float): density of MPCD particles inside sphere
+        R (float): Radius of Sphere 
+        kT (float): Temperature of MPCD particles (in energy units)
+        seed (int): Random seed for initialization
+
+    Returns:
+        Initialized MPCD system data inside a sphere (:py:class:`hoomd.mpcd.data.system`)
+
+    MPCD particles are randomly initialized inside the sphere.
+    An MPCD system can be randomly initialized only **after** the HOOMD system
+    is first initialized (see :py:mod:`hoomd.init`). The system can only be
+    initialized one time. The total number of particles *N* is evenly divided
+    between all domains. Random positions are then drawn uniformly within the
+    (local) box. Particle velocities are drawn from a Maxwell-Boltzmann
+    distribution consistent with temperature *kT*. All MPCD particles are given
+    unit mass and type A.
```

Comment:
This comment is out of date:
```suggestion
    initialized one time. Random particle positions are drawn uniformly in
    the sphere of radius *R* using rejection sampling within the box. Hence,
    the number of particles initialized may not be equal to :math:`4 \pi R^3 \rho/3`,
    where :math: `\rho` is the *density*. Particle velocities are drawn from a
    Maxwell-Boltzmann distribution consistent with temperature *kT*. All MPCD
    particles are given unit mass and type A.
```